src/app/room/components/room.component.html

```diff
@@ -1,23 +1,19 @@
 <copy-link></copy-link>
 
-<ng-container *ngIf="currentPlayer$ | async as storePlayer">
-  <div>Store Player information:</div>
-  <div *ngIf="storePlayer">
-    <div>Id: {{storePlayer.id}}</div>
-  </div>
-</ng-container>
-
-<ng-container *ngIf="firebasePlayer$ | async as firebasePlayer">
+<div *ngIf="firebasePlayer$ | async as firebasePlayer" class="top-spacer">
   <player [player]="firebasePlayer"></player>
-</ng-container>
+</div>
 <choice></choice>
 
 <div class="top-spacer">
   <opponents></opponents>
 </div>
 
 <spinner [loadingStatus]="loadingStatus">
-  <button mat-raised-button (click)="leaveRoom()" class="top-spacer">Leave the room</button>
+  <div class="top-spacer room-button-container">
+    <button mat-raised-button (click)="leaveRoom()">Leave the room</button>
+    <button mat-raised-button color="accent" (click)="startOver()">Restart the Game</button>
```

Comment:
Maybe disable it until there is a `lastOneActive`/every player is observer? otherwise you can restart anytime, even if you are out of the game.

Reply:
Hmm, but the players can decide to start over in the middle of the game because of some reason too (maybe something went wrong). They don't have to wait until everyone is done for many rounds to start over, in that case it will be easier to create another room IMO

Reply:
But then at least get confirmation from the other players or something. 
Like this, everyone can always restart when he is set inactive, and stop the others from continuing. this can't be correct no?

Reply:
1- That is why I didn't like this "start button" thing, but I chose to do it just because it helps to test :)
2- If we want the consent of everyone that is whole another "big need to be refined" use case.
3- I stick with the position that we should not just wait to everyone is done, cos it is confusing and it easier and cleaer to create a new room.

How should we proceed?